test/sharness/t0114-gateway-subdomains.sh

```diff
@@ -480,6 +480,66 @@ test_hostname_gateway_response_should_contain \
   "http://127.0.0.1:$GWAY_PORT" \
   "$CID_VAL"
 
+## Test subdomain handling of CIDs that do not fit in a single DNS Label (>63chars)
+## https://github.com/ipfs/go-ipfs/issues/7318
+## ============================================================================
+
+# TODO: replace with cidv1
```

Comment:
Right now `ipfs key gen -t ed25519 ed25519-test-key` outputs libp2p-key as Base58btc Multihash (thats why we hardcoded `12D..` below and not `bafy..`).

It is unclear if go-ipfs will switch text representation of libp2p-keys to CIDv1 in Base36 as part of switch to ed25519 (https://github.com/ipfs/go-ipfs/issues/6916), so for now we test Base58btc.